src/makeRobot.js

```diff
@@ -38,7 +38,61 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    info: `name: ${name}, chip version: ${version}, wheels: ${wheels}`,
+    coords:
+  {
+    x: 0, y: 0,
+  },
+
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward: function(steps = 1) {
+      this.coords.y += steps > 0 ? steps : 0;
+
```

Comment:
Check if steps value more than 0 than change value. Fix other methods too
```suggestion
    goForward: function(steps = 1) {
      if (steps > 0) {
        this.coords.y += steps;
      }

```